runtime/manifest-parser.peg

```diff
@@ -72,8 +72,23 @@ ManifestItem
   / ManifestStorage
   / Shape
   / Meta
+  / Resource
 
-Annotation = '@' annotation: lowerIdent { return annotation; }
+Annotation = '@' annotation:lowerIdent { return annotation; }
+
+Resource = 'resource' whiteSpace name:TopLevelIdent eolWhiteSpace Indent SameIndent 'start' eolWhiteSpace body:ResourceBody {
+  return {
+    kind: 'resource',
+    name,
+    data: body
```

Comment:
hmm, this should include `location`, `checkNormal` is supposed to error on this being absent. Looks like this change accidentally added 'object': https://github.com/PolymerLabs/arcs/pull/575/files#diff-b8b73ba247e6fb5f18a68ca683419175R27

Reply:
Reverted that and fixed the failing instances.